packages/bot/src/assets/sass/base/common.scss

```diff
@@ -1,9 +1,3 @@
-@at-root {
-    @import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&subset=latin,vietnamese,cyrillic-ext,latin-ext,cyrillic');
```

Comment:
Where are we importing the fonts from instead?

Reply:
@zaki-binary in shared in `google-fonts.scss`